.github/workflows/checkimage.yaml

```diff
@@ -0,0 +1,30 @@
+name: Regular base image update check
+on:
+  schedule:
+    - cron: "5 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install skopeo
+        run: sudo apt-get install -y skopeo
+      - name: Check change
+        run: |
+          skopeo inspect "docker://$(grep -Po '(?<=FROM )([^"]+)' Dockerfile)" | grep -Po '(?<="Digest": ")([^"]+)' > .baseimagedigest
+          docker run --rm --entrypoint sh -u 0 quay.io/cloudservices/drift-backend:latest -c \
+            'yum upgrade -y --security > /dev/null; rpm -qa | sort | sha256sum' \
```

Comment:
You can drop the `--security`.

Reply:
I used the same command as we have in `Dockerfile`